GOVERNANCE.md

```diff
@@ -65,109 +66,109 @@ including:
+by the TSC by logging a GitHub issue. Any Collaborator, TSC member, or the
+meeting chair can bring the issue to the TSC's attention by applying the
+`tsc-review` label. If consensus-seeking among TSC members fails for a
+particular issue, it may be added to the TSC meeting agenda by adding the
+`tsc-agenda` label.
+
+Prior to each TSC meeting, the meeting chair will share the agenda with
+members of the TSC. TSC members can also add items to the agenda at the
+beginning of each meeting. The meeting chair and the TSC cannot veto or remove
 items.
 
-The CTC may invite persons or representatives from certain projects to
+The TSC may invite persons or representatives from certain projects to
 participate in a non-voting capacity.
 
 The meeting chair is responsible for ensuring that minutes are taken and that a
 pull request with the minutes is submitted after the meeting.
 
 Due to the challenges of scheduling a global meeting with participants in
-several timezones, the CTC will seek to resolve as many agenda items as possible
+several timezones, the TSC will seek to resolve as many agenda items as possible
 outside of meetings using
-[the CTC issue tracker](https://github.com/nodejs/CTC/issues). The process in
+[the TSC issue tracker](https://github.com/nodejs/TSC/issues). The process in
 the issue tracker is:
 
-* A CTC member opens an issue explaining the proposal/issue and @-mentions
-  @nodejs/ctc.
-* After 72 hours, if there are two or more `LGTM`s from other CTC members and no
-  explicit opposition from other CTC members, then the proposal is approved.
-* If there are any CTC members objecting, then a conversation ensues until
+* A TSC member opens an issue explaining the proposal/issue and @-mentions
+  @nodejs/tsc.
+* After 72 hours, if there are two or more `LGTM`s from other TSC members and no
+  explicit opposition from other TSC members, then the proposal is approved.
+* If there are any TSC members objecting, then a conversation ensues until
   either the proposal is dropped or the objecting members are persuaded. If
   there is an extended impasse, a motion for a vote may be made.
 
 ## Consensus Seeking Process
 
-The CTC follows a
+The TSC follows a
 [Consensus Seeking](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
 decision making model.
 
 When an agenda item has appeared to reach a consensus, the meeting chair will
 ask "Does anyone object?" as a final call for dissent from the consensus.
 
-If an agenda item cannot reach a consensus, a CTC member can call for either a
+If an agenda item cannot reach a consensus, a TSC member can call for either a
 closing vote or a vote to table the issue to the next meeting. All votes
-(including votes to close or table) pass if and only if more than 50% of the CTC
+(including votes to close or table) pass if and only if more than 50% of the TSC
 members (excluding individuals who explicitly abstain) vote in favor. For
```

Comment:
Is this in line with the TSC charter? I am not sure if I understand https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-voting, but it was mentioned yesterday that any abstention could be counted as a vote against the resolution here.

I would like us to keep the behavior defined in this (CTC) document, though, where explitic abstaintions are subtracted from the total number of people voting.